test_ruff.py

```
@@ -0,0 +1,301 @@
+# Ruff-specific rules (RUF) - catches bugs and errors
+def test_ruff():
+    # RUF001: String contains ambiguous unicode character
+    text = "Hello\u2028World"  # RUF001
```

Ruff (0.12.2)

Local variable `text` is assigned to but never used

Remove assignment to unused variable `text`

(F841)